### PR TITLE
Fix accelerator key conflicts

### PR DESCRIPTION
**Pull request by softins**
CHANGELOG: Fix some accelerator key conflicts, and fix ellipses in Norwegian

**Context: Fixes an issue?**

Conflicts discovered as part of the pre-release checks for 3.10.0 using `tools/checkkeys.pl`

**Does this change need documentation? What needs to be documented and how?**

No

**Status of this Pull Request**

Ready for merge

**What is missing until this pull request can be merged?**

Nothing, only maintainer approval

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#source-code-consistency) 
-  [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [x] I've filled all the content above

### CODE REVIEW

File: src/translation/translation_nb_NO.ts [hunk 1]
```diff
@@ -736,17 +736,17 @@
     <message>
         <location filename="../clientdlg.cpp" line="286"/>
         <source>&amp;Connection Setup...</source>
-        <translation>&amp;Tilkoblingsoppsett …</translation>
+        <translation>&amp;Tilkoblingsoppsett...</translation>
     </message>
     <message>
         <location filename="../clientdlg.cpp" line="290"/>
         <source>&amp;Load Mixer Channels Setup...</source>
-        <translation type="unfinished">&amp;Last inn kanalmikserinnstillingene …</translation>
```
Comment (ann0see) [inline src/translation/translation_nb_NO.ts:744]: I believe the initial ... was typographically better.

**Review by ann0see — APPROVED** [review]
Thanks!

File: src/translation/translation_nb_NO.ts (hunk 1 above)
Comment (pljones) [reply src/translation/translation_nb_NO.ts:744]: UK keyboards do not have that character.  I've no idea what UTF-8 character it is.  Ideally, the character would be coded as `&whatever;` so it was easy to see (like `&amp;`).

Comment (softins) [reply src/translation/translation_nb_NO.ts:744]: In Jamulus, both English and all the other translations use a plain three dots `...`, so I changed nb_NO to the same for consistency.

Also, in the font on my RPi, at least as viewed via VNC, the ellipsis character was very indistinct, looking just like a small underscore.

Just for reference, in reply to @pljones, the ellipsis is at position 0x85 in Windows-1252 encoding, can be input on a PC using Alt-0133, and in UTF-8 is code point `0x2026`. In HTML, it is `&hellip;` (for "horizontal ellipsis): &hellip;

**Review by pljones — APPROVED** [review]

**Pull request merged**